anthos-vmruntime/README.md

```diff
@@ -158,7 +158,7 @@ create the VM using `kubectl apply -f pos-vm.yaml`.
 ```sh
 kubectl virt create vm pos-vm \
 --boot-disk-size=80Gi \
---boot-disk-storage-class=standard \
+--boot-disk-storage-class=local-shared \
```

Comment:
Can we remove it completely? 
Because the default is already `local-shared`. So we won't this line at all